src/app.js

```diff
@@ -0,0 +1,111 @@
+import axios from "axios";
+async function findCountry(){
+    const BASE_URI = 'https://restcountries.com/'
+    const ENDPOINT = 'v2/name/netherlands'
```

Comment:
Variabelen camelCase schrijven
```suggestion
    const base_uri = 'https://restcountries.com/'
    const endpoint = 'v2/name/netherlands'
```

Reply:
ok